src/test/java/org/openmainframeproject/cobolcheck/MockIT.java

```diff
@@ -353,10 +352,25 @@ private String removeBoilerPlateCode(String code, List<String> boilerPlateTags){
                     "                   ALSO ANY                                                     " + Constants.NEWLINE +
                     "                    PERFORM UT-1-0-1-MOCK                                          " + Constants.NEWLINE +
                     "           WHEN OTHER                                                           " + Constants.NEWLINE +
+                    "                    PERFORM UT-1-0-0-WO                                           " + Constants.NEWLINE +
+                    "            END-EVALUATE                                                         " + Constants.NEWLINE +
+                    "           .                                                                   " + Constants.NEWLINE +
+                    "                                                                              " + Constants.NEWLINE +
+                    "      *****************************************************************     " + Constants.NEWLINE +
+                    "      *WhenOther block called                                        " + Constants.NEWLINE +
+                    "      *****************************************************************     " + Constants.NEWLINE +
+                    "       UT-1-0-0-WO SECTION.                                                 " + Constants.NEWLINE +
+                    "      *****************************************************************     " + Constants.NEWLINE +
+                    "      *When other block of: SECTION: 000-START                                        " + Constants.NEWLINE +
+                    "      *In testsuite: \"Mocking tests\"                                        " + Constants.NEWLINE +
```

Comment:
Testsuite and testcase is not interesting information in the context of a WhenOther section/paragraph, as there will at most be one WhenOther for each section/paragraph. The testsuite and -case is just informing which test it belongs to, which is valuable info for a mock, but a WhenOther only needs to inform about the section/paragraph, that it comes from 😄